### PR TITLE
Esthetic Adjustments

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/5496

Live Demo:
https://5496-sg-fileupload-changes--atd-moped-main.netlify.app/moped/session/signin

### CODE REVIEW

**Review by amenity — CHANGES_REQUESTED** [review]
See additional tweaks in https://github.com/cityofaustin/atd-data-tech/issues/5496.

File: moped-editor/src/utils/humanReadableFileSize.js [hunk 1]
```diff
@@ -16,8 +15,8 @@ export default (bytes, si=false, dp=1) => {
     }
 
     const units = si
-        ? ['kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
-        : ['KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
+        ? ['KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+        : ['KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```
Comment (mateoclarke) [inline moped-editor/src/utils/humanReadableFileSize.js:19]: I'm confused what's happening here now. Both conditions look identical.

**Review by mateoclarke — COMMENTED** [review]
Adding my 👍  but I'm unsure of the status so going to defer to @amenity for final approval. Most of these small updates make sense to me tho.

File: moped-editor/src/utils/humanReadableFileSize.js (hunk 1 above)
Comment (sergiogcx) [reply moped-editor/src/utils/humanReadableFileSize.js]: Ah, good call. Yea they are now identical. I've removed the two lists and we only have one now.

**Review by mddilley — APPROVED** [review]
🙏

**Pull request merged**